docs/running-on-kubernetes.md

```diff
@@ -1811,6 +1811,50 @@ spec:
   queue: default
 ```
 
+#### Using Apache YuniKorn as Customized Scheduler for Spark on Kubernetes
+
+[Apache YuniKorn](https://yunikorn.apache.org/) is a resource scheduler for Kubernetes that provides advanced batch scheduling
+capabilities, such as job queuing, resource fairness, min/max queue capacity and flexible job ordering policies.
+For available Apache YuniKorn features, please refer to [this doc](https://yunikorn.apache.org/docs/next/get_started/core_features).
+
+##### Prerequisites
+
+Install Apache YuniKorn:
+
+```bash
+helm repo add yunikorn https://apache.github.io/yunikorn-release
+helm repo update
+kubectl create namespace yunikorn
+helm install yunikorn yunikorn/yunikorn --namespace yunikorn
```

Comment:
It seems that `YuniKorn` helm chart doesn't have `TEST SUITE`. Did I understand correctly?
```
$ helm install yunikorn yunikorn/yunikorn --namespace yunikorn
NAME: yunikorn
LAST DEPLOYED: Tue Aug 23 08:03:37 2022
NAMESPACE: yunikorn
STATUS: deployed
REVISION: 1
TEST SUITE: None
```

Reply:
BTW, IIRC, Apache YuniKorn is now TLP project, not an incubation project.
- https://apache.github.io/yunikorn-release

Could you update Apache YuniKorn link please?

<img width="844" alt="Screen Shot 2022-08-23 at 9 45 07 AM" src="https://user-images.githubusercontent.com/9700541/186215061-58bbefbd-90ae-4bad-897e-bb51581246c6.png">
/

Reply:
Good catch, let me fix that.

Reply:
this page https://apache.github.io/yunikorn-release has been fixed.

> It seems that YuniKorn helm chart doesn't have TEST SUITE.

Yes, there is no test suite today in the helm charts.

Reply:
Thanks!